packages/cli/src/util/dev/server.ts

```diff
@@ -2319,7 +2335,7 @@ export default class DevServer {
 
     this.output.debug(`Spawning dev command: ${command}`);
 
-    const devPort = new URL(this.address).port;
+    const devPort = this.addressURL.port;
```

Comment:
Same here, you can remove `devPort`

Reply:
Good call: [`ae1e1ee` (#8538)](https://github.com/vercel/vercel/pull/8538/commits/ae1e1ee0b3f9d68ebbf71e0036e742d81844c1de)